docs/en/writing-running-appium/caps.md

```diff
@@ -167,6 +167,7 @@ These Capabilities are available only on the [Espresso Driver](/docs/en/drivers/
 |`espressoServerLaunchTimeout`|Timeout in milliseconds used to wait for the Espresso server to launch. Defaults to `30000` |e.g., `50000`|
 |`espressoBuildConfig`|Path to the Espresso server build configuration JSON (see below)|e.g., `/projects/myapp-tests/buildconfig.json`|
 |`showGradleLog`|Whether to pipe Gradle build log for the Espresso server to the Appium log. Defaults to `false`|e.g., `true`|
+|`skipServerInstallation`|Skip Espresso server build and apk installation. This option could break proper Espresso server setup for particular Appium version, but it can improve startup performance when the proper Espresso server and the proper app under test are already installed on the device. So, please consider to use this option after updating espresso related environment wihtout this option if you would like to use this. Defaults to `false` | `true` or `false`|
```

Comment:
Please, make sure not to enable this option if the Espresso server or the application under test need an update